Core/Utilities/TimeLeft/test/Test_TimeLeft.py

```diff
@@ -53,13 +93,27 @@ def test_getScaledCPU( self ):
       self.tl.runCommand = rcMock
 
       batchSystemName = '%sTimeLeft' % batch
-      batchPlugin = __import__( 'DIRAC.Core.Utilities.TimeLeft.%s' %
+      batchPlugin = __import__( 'DIRAC.Core.Utilities.TimeLeft.%s' % #pylint: disable=unused-variable
                                 batchSystemName, globals(), locals(), [batchSystemName] )
       batchStr = 'batchPlugin.%s()' % ( batchSystemName )
       tl.batchPlugin = eval( batchStr )
       res = tl.getScaledCPU()
       self.assertEqual( res, 300.0 )
 
+      for batch, retValue in [( 'MJF', MJF_ReturnValue )]:
+        self.tl = importlib.import_module( "DIRAC.Core.Utilities.TimeLeft.TimeLeft" )
+        rcMock = MagicMock()
```

Comment:
I think you can do that with the simple @mock.patch decorator, isn't it ?  I find it more readable, but maybe it is a personal taste